### PR TITLE
Add method to get featured content by date

### LINKED ISSUE #34: Implement featured content api
There are a lot of new REST APIs for wikipedia present in the REST API docs.

We'll look through them one by one and implement the same. Anyone who wants to pick up on any other REST API should ideally open a new issue.

The REST API has a /feed/featured/{year}/{mm}/{dd} endpoint which provides featured content for that particular day. Implementation for this can follow #8 .

Implementation for this should follow the summary or related method flow. Remember to write unit tests for all possible scenarios in your new functions, and try to use types as far as possible.

### PR DESCRIPTION
**Pull request by friendofdog**
resolves #34

### CODE REVIEW

**Review by dopecodez — APPROVED** [review]

**Comment by friendofdog** [comment]
@dopecodez Thanks for approving my PR.
I can't merge this as I don't have write access.

**Comment by dopecodez** [comment]
Sorry @friendofdog , i was busy with my marriage for a month! I'm back and lets keep Wikipedia up to date.
Thanks for your contribution 🥇

**Comment by friendofdog** [comment]
@dopecodez 
No worries (and congrats!)

**Pull request merged**
